ozw-network-visualization-card.js

```diff
@@ -141,11 +142,15 @@ class OZWNetworkVisualizationCard extends HTMLElement {
   _buildLabel(device) {
     var regDevice = this.device_registry[device.node_id];
 
-    var res = regDevice ? "<b>" + regDevice.name + "</b>\n" : "";
+    //Add user's device name for display
+    var res = regDevice ? "<b>" + regDevice.name_by_user + "</b>\n" : "";
```

Comment:
If a user defines the names in OZW, they will be in `regDevice.name`. Also, if they don't set a name in HA, `name_by_user` is null. Fix:
```suggestion
    var res = regDevice ? "<b>" + (regDevice.name_by_user || regDevice.name) + "</b>\n" : "";
```

Reply:
Good suggestion. Again, how do I change this?